### PR TITLE
Fix url to test runner interface docs

### PR DESCRIPTION
**Pull request by OranShuster**

### CODE REVIEW

**Review by jmrunkle — APPROVED** [review]
Thanks, just saw the same issue myself

**Comment by iHiD** [comment]
Please could this link to the website: https://exercism.org/docs/building/tooling/test-runners/interface

The docs repo isn't stable in terms of URLs and isn't intended to be directly linked to.

File: README.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ To run a solution's tests using a Docker container, follow these steps:
    1. Once the script has completed, the test results will be written to `<output-directory>/results.json`.
 
 
-[test-runner-interface]: https://github.com/exercism/automated-tests/blob/master/docs/interface.md
\ No newline at end of file
+[test-runner-interface]: https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md
```
Comment (jmrunkle) [inline README.md:28]: Copying the comment directly in a review:

> Please could this link to the website: https://exercism.org/docs/building/tooling/test-runners/interface
>
>The docs repo isn't stable in terms of URLs and isn't intended to be directly linked to.

**Review by jmrunkle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (OranShuster) [reply README.md:28]: so in general - all markdown links should point to their "deployed" web counterparts?

Comment (jmrunkle) [reply README.md]: I think that is a "wherever possible" thing. Not all markdown files inside repos have deployed counterparts on the webpage.

**Review by jmrunkle — APPROVED** [review]

**Comment by iHiD** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (SleeplessByte) [reply README.md]: Moreso the docs don't actually allow traversing links on GitHub, because their paths aren't correct for GitHub.

**Pull request merged**